content/docs/authorization/cli.md

```diff
@@ -206,8 +193,9 @@ karavictl generate [flags]
 ##### Options inherited from parent commands
```

Comment:
Should this heading be changed throughout the file to clarify that some of these flags are required? This seems to be a source of confusion from the bug

Reply:
Made changes to specify the Required and Optional flags for each command.